app/router/router.go

```diff
@@ -40,6 +42,16 @@ func encodeResponse(w http.ResponseWriter, statusCode int, body interface{}) {
 	}
 }
 
+func encodeResponseJson(w http.ResponseWriter, statusCode int, body interface{}) {
+	w.Header().Set("x-amzn-RequestId", "00000000-0000-0000-0000-000000000000")
+	w.Header().Set("Content-Type", "application/x-amz-json-1.0")
+	w.WriteHeader(statusCode)
+	enc := json.NewEncoder(w)
+	if err := enc.Encode(body.(app.CreateQueueResponse).Result); err != nil {
```

Comment:
I was convinced that it was necessary to separate the protocol description from `gosqs.go` (also on `gosns.go`)...